bpf/bpf_lxc.c

```diff
@@ -812,6 +818,17 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 skip_egress_gateway:
 #endif
 
+#ifdef ENABLE_EXTERNAL_VXLAN
+	{
+		if (!vtep_mac || !tunnel_endpoint)
+			goto skip_external_vxlan;
+		if ( eth_store_daddr(ctx, (__u8 *)&vtep_mac, 0) < 0)
+			return DROP_WRITE_ERROR;
+		return __encap_and_redirect_with_nodeid(ctx, tunnel_endpoint, WORLD_ID, monitor);
+	}
+skip_external_vxlan:
```

Comment:
nit: unused label?

Reply:
`skip_external_vxlan`  when `if (!vtep_mac || !tunnel_endpoint)`, am I  miss anything :)

Reply:
No, it's me who missed it. I saw the red `goto` statement and didn't read closely enough to see that it was a `goto` rather than a `return`. LGTM :+1: